### DISCUSSION TITLE
how can i upload my project file and host on github pages .

### DISCUSSION THREAD

**Discussion by BLACKHOODIE-pro** [community/community#143502] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

give me some ideas

**Comment by Sikorsky3301** [comment]
go to the your settings on your repo

**Comment by Sikorsky3301** [comment]
hope that helps

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/143503#discussioncomment-11145416) for reference to why this discussion was closed and locked. Thanks!
